source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
+* 50 requests against different items with Data_Type other than Book.
+* 10 requests against different items within each of 5 books, for 50 total requests.
 
-The auditor must carry out 15 **inside** tests.
+If a platform only delivers book content as a whole-book download (i.e. without Book_Segments), then each request SHOULD be for a separate book.
 
-This must result in:
+The auditor activity MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 5 Unique_Title_Requests (1 per book).
 
-* 15 Total_Item_Requests being reported in the DR_D1 Standard View.
+**Option 2**: Platforms that only include Data_Type=Book.
 
-The auditor must carry out 15 **outside** tests.
+The auditor MUST make a total of 100 requests
 
-This must result in:
+* 10 requests against different items within each of 10 books, for 100 total requests.
 
-*  30 Total_Item_Requests being reported in the DR_D1 Standard View.
+If a platform only delivers book content as a whole-book download (i.e. without Book_Segments), then each request SHOULD be for a separate book.
 
+The auditor activity MUST result in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 10 Unique_Title_Requests (1 per book).
 
-Audit Test D1-4: Total_Item_Investigations
-..........................................
+**Option 3**: Platforms that do not include content with Data_Type=Book.
 
-**IMPORTANT NOTE**: This test is required when investigations can be reported independently of a request. It is not required if all investigations have a matching request, but this must be verified during the audit. Any exclusion of tests must be confirmed by COUNTER prior to testing and the auditor notified.
+The auditor MUST make a total of 100 requests against different items, resulting in 100 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests. There should be 0 Unique_Title_Investigations and 0 Unique_Title_Requests.
 
-The auditor must make 100 Investigations on a subset of available unique items. This must result in 100 Total_Item_Investigations.
 
-Multiple paths should be used to make the Investigations. When possible, 50% of items Investigations should be via browsing and 50% via searching.
+E.2.3 Database Report Audit Tests
+"""""""""""""""""""""""""""""""""
 
-If either browsing to item investigations or accessing item investigations via searching is not possible, then 100% of item investigations can be made via the only available option.
+The Database Report will be COUNTER-compliant if the following audit tests are passed. The Database Search and Item Usage (DR_D1) and Database Access Denied (DR_D2) Standard Views of the Database Report will be COUNTER-compliant if the metrics match those in the Database Report, with the appropriate aggregation.
```

Comment:
E.2.3 only mentions databases. I assume an eBook_Collection or Multimedia_Collection also is regarded as a database, and also a Discovery_Service (unless it consists of multiple databases like EDS)?